README.md

```diff
@@ -126,7 +128,7 @@ We also welcome translations for Wagtail's interface. Translation work should be
 
 ### License
 
-[BSD](https://github.com/wagtail/wagtail/blob/main/LICENSE)
+[BSD](https://github.com/wagtail/wagtail/blob/main/LICENSE) - Free to use "as is" for any purpose.
```

Comment:
This makes it sound like you're not allowed to modify it. How about: "free to use and modify for any purpose, including both open and closed-source code"?